### ISSUE TITLE
How is this implemented in a web API

### ISSUE THREAD

**Issue by JimF42** [MosheWorld/AuthenticationService#3]
I have been looking for something like this for a new RESTful service I am writing (I'm new to web APIs, but not to other c# programming).

What I didn't glean from the code or your article is how to I implement this into a Web API application. I am using VS 2022 and Core 6. I presume it would get wired into the Startup's ConfigureServices, but I don't see how that's done.

Is it possible to provide a snip to show us?

**Comment by Rekasi** [comment]
@JimF42 Did you find a way to implement it yourself? Best Regards.

**Comment by JimF42** [comment]
@Rekasi, No, I didn't. We ended up not going with a web service for this internal only app.

**Comment by MosheWorld** [comment]
@JimF42 Yes, there you go.
[https://github.com/MosheWorld/Microservices-NodeJS](https://github.com/MosheWorld/Microservices-NodeJS)

Please review the signup, signin, and validation methods in the User Microservice. Then, examine how the middleware is implemented for validation in the [Product route](https://github.com/MosheWorld/Microservices-NodeJS/blob/master/apps/product/src/routes/product-route.ts).

**Comment by MosheGenocide** [comment]
this is a comment for Moshe 
![image](https://github.com/user-attachments/assets/13436026-ba2b-4959-9f30-ab6dd693a170)
